src/components/EditableField.jsx

```diff
@@ -42,6 +42,7 @@ const EditableField = (props) => {
   }, [editing]);
 
   const deleteEditableField = (e) => {
+    e.stopPropagation();
```

Comment:
Why do we need to add `e.stopPropagation` here?

Reply:
All the other actions in `EditableField` have this `e.stopPropagation`, you can check `onSubmit` and `onCancel`.

From a more practical standpoint, if I remove the `e.stopPropagation` from `onDelete` then when I delete a cellSet then that `click` event propagates up to the `onCheck` (triggered by antd on `<Tree>`).